package.json

```diff
@@ -5,14 +5,14 @@
   "license": "MIT",
   "repository": "chalk/ansi-styles",
   "author": {
-    "name": "Sindre Sorhus",
     "email": "sindresorhus@gmail.com",
+    "name": "Sindre Sorhus",
     "url": "sindresorhus.com"
   },
   "maintainers": [
-    "Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)",
+    "JD Ballard <i.am.qix@gmail.com> (github.com/qix-)",
     "Joshua Appelman <jappelman@xebia.com> (jbnicolai.com)",
-    "JD Ballard <i.am.qix@gmail.com> (github.com/qix-)"
+    "Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)"
   ],
```

Comment:
Not sure why this changed. I didn't (consciously) touch it; might have been NPM. I'll fix it.